### PR TITLE
Feature/migrate to smart wallet UX improvements menu & banner

### PR DESCRIPTION
**Pull request by mdjastrzebski**
Scope:
- [x] Migrate to smart wallet banner
- [x] Light refactor to Menu screen

### CODE REVIEW

File: src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  return (
+    <TouchableOpacity onPress={onPress} style={style}>
+      <BackgroundGradient colors={GRADIENT_COLORS} locations={[0.05, 0.65]} useAngle angle={171}>
+        <Icon source={smartWalletIcon} />
+
+        <Summary>
+          <Title>{t('title')}</Title>
+
+          <Spacing h={spacing.small} />
+
```
Comment (jdnmr) [inline src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js:56]: Why the empty lines?

File: src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+`;
+
+const Summary = styled.View`
+  flex: 1;
+`;
+
+const Title = styled(Text)`
+  font-family: '${appFont.medium}';
+  ${fontStyles.big};
+  color: #fcfdff;
```
Comment (jdnmr) [inline src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js:84]: We have this color exported from our theme/colors modules

**Review by jdnmr — APPROVED** [review]

File: src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js (hunk 1 above)
Comment (mdjastrzebski) [reply src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js:56]: IMO improves readability.

File: src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js (hunk 2 above)
Comment (mdjastrzebski) [reply src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js:80]: This color is actually present only in dark theme, while the banner should look the same in both light and dark mode.

File: src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js (hunk 1 above)
Comment (mdjastrzebski) [reply src/components/Banners/MigrateWalletBanner/MigrateWalletBanner.js]: Fixed

**Pull request merged**